Sources/TensorFlow/Core/Utilities.swift

```diff
@@ -167,3 +167,176 @@ extension UnsafeMutablePointer where Pointee == CTensorHandle? {
         self.init(unwrapped)
     }
 }
+
+//===------------------------------------------------------------------------------------------===//
+// Hashing
+//===------------------------------------------------------------------------------------------===//
+
+internal extension FixedWidthInteger {
+    @usableFromInline
+    init(fromBytes bytes: ArraySlice<UInt8>, startingAt index: Int) {
```

Comment:
Per Swift API Design Guidelines, the base name of an initializer (aka the type name) should not form a phrase with argument labels. 
```suggestion
    init(bytes: ArraySlice<UInt8>, startIndex: Int) {
```

Reply:
Sorry I keep forgetting that.